dd-sdk-android/src/test/kotlin/com/datadog/android/v2/core/DatadogCoreTest.kt

```diff
@@ -286,7 +286,6 @@ internal class DatadogCoreTest {
         assertThat(testedCore.webViewRumFeature).isNull()
         assertThat(testedCore.crashReportsFeature).isNull()
         assertThat(testedCore.sessionReplayFeature).isNull()
-        assertThat(testedCore.contextProvider).isNull()
```

Comment:
why this line is removed? In the latest version of the SDK v2 branch at least when `CoreFeature` is stopped, `contextProvider` should return `null`.